theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer.tsx

```diff
@@ -91,19 +91,22 @@ export class TraceViewerWidget extends ReactWidget implements StatefulWidget {
         this.title.closable = true;
         this.addClass('theia-trace-open');
         this.backgroundTheme = ThemeService.get().getCurrentTheme().type;
-        ThemeService.get().onDidColorThemeChange(() => this.updateBackgroundTheme());
+        this.toDispose.push(ThemeService.get().onDidColorThemeChange(() => this.updateBackgroundTheme()));
```

Comment:
Since the upgrade to Theia 1.28, the API of the theme service has changed. This line needs to be:
`this.toDispose.push(this.themeService.onDidColorThemeChange(() => this.updateBackgroundTheme()));`

Please rebase this PR and update the PR.